docs/roadmap.markdown

```diff
@@ -4,9 +4,122 @@ title: "Roadmap"
+
+### 2020 Spring
+
+| State | Tool | Milestone | Rationale |
+| ----- | ---- | --------- | --------- |
+| Planning | New Relic Client | GraphQL Support | NerdGraph, New Relic's implementation of GraphQL is the future platform for all API requests, and is the backing solution for New Relic One. As such, the NR Client needs to implement GraphQL to support any additional data. |
+| Planning | New Relic Terraform Provider | Provider 2.0 | In order to add some newer features to the New Relic Terraform Provider, we need to implement some breaking changes.  This is a delicate area, so reving the major version and implementing the most important of those changes is warranted. |
+| Planning | New Relic CLI | Initial Support | Community members with existing workflows want to be able to leverage command line tooling for interacting with New Relic. Leveraging the New Relic Client, we can build a CLI to simplify user interactions with New Relic in scripted and interactive workflows. |
+| Planning | CloudFormation | Initial Support | New Relic has released a minimal Partner integration for CloudFormation specifically covering NRQL Alerting. Evaluate if reproducing this using the New Relic Client would allow for quickly adding support for other resources available through the NR Client for feature parity with the New Relic Terraform Provider. |
+| Planning | Developer Toolkit Docs | Enhanced Docs and Examples | The Developer Toolkit as a collection of tools is extremely useful, however spreading the documentation across multiple repositories and in various levels of detail increases friction.  Create a consolidated collection of examples and documentation for tools, including when it is appropriate to use a specific tool vs another. |
+
+### 2020 Summer
+
+| State | Tool | Milestone | Rationale |
+| ----- | ---- | --------- | --------- |
+| Proposed | New Relic Client | Lambda Support | For community members already leveraging Serverless for their workflows, enabling the New Relic Client to be deployed in this way creates a quick path to embedding New Relic along side their existing best practices. |
+| Proposed | New Relic CLI | Streamlined Installation | Enabling easy installation and upgrading of the NR CLI following existing workflows provides a streamlined experience to the user.  This should include installation through well-known package managers for supported platforms, so maintenance of the CLI fits into existing practices for the user. |
+| Proposed | New Relic CLI | New Relic Logs Support | Integration of New Relic Logs and the NR CLI extends the user experience of the New Relic's logging solution into the terminal. |
+| Proposed | Developer Toolkit Docs | Curate Community Docs / Examples | As a community focused project, it is important that we work with the extended community in providing documentation and example workflows. Engaging in this way will drive inovation across the broader community, and easy onboarding for those new to the space. |
```

Comment:
change "extended community in providing" to "extended community to provide".  Also innovation is misspelled.  I don't think you need to reference "broader community" since you already refer to community engagement a few times before this.  Just "Engaging in this way will drive innovation and easy onboarding for those new to the space"